internal/alloydb/instance.go

```diff
@@ -125,6 +125,9 @@ func (r *refreshOperation) IsValid() bool {
 // required information approximately 5 minutes before the previous certificate
 // expires (every 55 minutes).
 type Instance struct {
+	// OpenConns is the number of open connections to the instance.
+	OpenConns uint64
```

Comment:
Can we move this commit into a seperate PR with a "fix:" title?

Reply:
done